packages/dates/src/calendar/Calendar.tsx

```diff
@@ -147,6 +152,7 @@ const useStyles = makeStyles<
       width: theme.spacing(5),
       height: theme.spacing(5),
       borderRadius: theme.spacing(0.5),
+      fontSize: '12px',
```

Comment:
Make sure we are using absolute values. Not relative ones.

Reply:
Yes I have checked all the component and everything works as expected. Here I have changed from `12px` to `12`.